bin/update-pot-file-references.php

```diff
@@ -89,7 +89,7 @@ function load_js_transpiling_source_maps(): array {
 		}
 
 		foreach ( $file_json[ 'sources' ] as $source ) {
-			$source = preg_replace( '%^webpack:///\./(client/.*)$%', '${1}', $source );
+			$source = preg_replace( '%^webpack://\./(client/.*)$%', '${1}', $source );
```

Comment:
In the Webpack 5 config, we are manually setting `devtoolModuleFilenameTemplate`, so we can replace the previous triple slash prefixes with normal URL-like paths `webpack://[resource-path]`